ios/PurchasesHybridCommon/PurchasesHybridCommon/PaywallProxy.swift

```diff
@@ -25,6 +25,11 @@ import UIKit
         return UIHostingController(rootView: PaywallView())
     }
 
+    @objc
+    public func createFooterPaywallView() -> UIViewController {
+        return PaywallFooterViewController()
```

Comment:
Maybe you're planning to add this in future PRs but do we need to pass in optional parameters like the offering?

Reply:
yes! I was planning on adding those whenever we add start adding the parameters to the hybrids, after we have the views fully working